### PR TITLE
CTCP-2333 Create gross mass amount page (on local branch CTCP-2333-2)

### PR DESCRIPTION
**Pull request by kenneth-rayner**

### CODE REVIEW

**Review by Reece-Carruthers — APPROVED** [review]

File: conf/messages.en [hunk 1]
```diff
@@ -63,7 +63,7 @@ dateGoodsUnloaded.heading = When were the goods unloaded?
 dateGoodsUnloaded.hint = For example, 14 1 2020
 dateGoodsUnloaded.checkYourAnswersLabel = When were the goods unloaded?
 dateGoodsUnloaded.error.required.all = Enter the date when the goods were unloaded
-dateGoodsUnloaded.error.required.two = The date goods were unloaded must include {0} and {1}
+dateGoodsUnloaded.error.required.two = The date goods were unloaded must include {0} and {1} me
```
Comment (vajarova) [inline conf/messages.en:66]: Is this a typo?

Comment (kenneth-rayner) [reply conf/messages.en]: fixed

File: conf/messages.en [hunk 2]
```diff
@@ -87,7 +87,8 @@ vehicleRegistrationCountry.placeHolder = Select a country
 
 grossWeight.title = What is the new gross weight of item {0}?
 grossWeight.heading = What is the new gross weight of item {0}?
-grossWeight.checkYourAnswersLabel = What is the new gross weight of item {0}?
+grossWeight.heading = What is the new gross weight of item {0}?
```
Comment (Reece-Carruthers) [inline conf/messages.en:90]: Have grossWeight.heading twice

Comment (kenneth-rayner) [reply conf/messages.en]: removed duplicate

**Review by Reece-Carruthers — APPROVED** [review]

**Review by vajarova — APPROVED** [review]

**Pull request merged**